src/index.mjs

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable no-process-exit */
 /* eslint no-console: 'off' */
 
+import * as path from 'node:path';
+import { cosmiconfigSync } from 'cosmiconfig';
 import detectPackageManager from 'which-pm-runs';
 import { execa } from 'execa';
 import fs from 'fs';
```

Comment:
[suggest] Does it make sense to use `node:` import for `fs` like `path`?

```suggestion
import * as fs from 'node:fs';
```

Reply:
Great point, I should be consistent across the files - will do!